build_extra/rust/BUILD.gn

```diff
@@ -32,6 +32,11 @@ rust_component("url") {
 
 rust_component("percent_encoding") {
   source_root = "$crates/url/percent_encoding/lib.rs"
+  args = [
+    # suppress some warnings
+    "-Aunused-imports",
+    "-Adeprecated",
+  ]
```

Comment:
Why does this module have these warnings? Do the authors not care (seems unlikely) or are we compiling it with the wrong cfg? Can you investigate further? If it's their issue, please open an issue with them and link to the issue here.

Reply:
They seem caused by the line `use std::ascii::AsciiExt;`. The module seems deprecated at recent version of rust. Their build of master branch also shows [the same warnings](https://travis-ci.org/servo/rust-url/jobs/400867523). So I think it's not a problem of our configuration.

I think this should be fixed in their repository. I filed an issue there. https://github.com/servo/rust-url/issues/455